scripts/buildwithfdpp.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+GITDIR=../fdpp.git
+
+# Notes:
+#    1/ Copy this script to the top directory of the dosemu source,
+#       as it won't exist in earlier versions and will be removed
+#       during bisect checkouts.
+#    2/ Change the above variable to match your path to FDPP git location.
+#    3/ This script has been tested with dosemu2 / fdpp combinations to
+#       API 26..13 tags okay to configure, build and run FDPP plugin.
+#       API 12..11 tags okay to configure, build and run but with FS issues
+#       API  <= 10 tags not tested
+#
+#    Usage: ./buildwithfdpp.sh --matchapi
+
+if [ ! -d src ] ; then
+  echo Run this script from the top directory of the dosemu2 source
+  exit 3
+fi
+
+GITDIR=$(realpath ${GITDIR})
+if [ ! -d ${GITDIR} ] ; then
+  echo Local FDPP git directory \"${GITDIR}\" does\'t exist
+  exit 2
+fi
+
+FDPP_PLUGIN_SOURCE=src/plugin/fdpp/fdpp.c
+if [ ! -f ${FDPP_PLUGIN_SOURCE} ] ; then
+  echo FDPP plugin was not implemented at this version of Dosemu2
+  exit 1
+fi
+
+FDPP_PLUGIN_VER=$(awk -F= '/FDPP_API_VER !=/{printf("%d", $2);}' ${FDPP_PLUGIN_SOURCE})
+FDPP_LIBRARY_VER=$(awk -F" " '/#define FDPP_API_VER/{printf("%d", $3);}' "${GITDIR}/fdpp/thunks.h")
+
+if [ ! "${FDPP_PLUGIN_VER}" = "${FDPP_LIBRARY_VER}" ] ; then
+  if [ ! "$1" = "--matchapi" ] ; then
+    echo FDPP version mismatch in "${GITDIR}"
+    echo need API"${FDPP_PLUGIN_VER}" but have API"${FDPP_LIBRARY_VER}"
+    exit 1
+  fi
+  echo Building FDPP at API${FDPP_PLUGIN_VER}
+  (cd ${GITDIR} && git checkout -f API${FDPP_PLUGIN_VER} && make clean all)
+fi
+
+[ -d pkgconfig ] || mkdir pkgconfig
```

Comment:
You can now do `./default-configure fdpp-build-path=/home/stas/src/fdpp/fdpp/build`
Could you please adapt?

Reply:
But if I do that it won't be able to help bisect anything before you added that option.

Reply:
OK, how about grepping `default-configure` for
`fdpp-build-path` and use it if its there, otherwise
do what you do now? One day we'll just remove
that part.

Reply:
okay